src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/RSBackupVaultCmdletBase.cs

```diff
@@ -17,32 +18,44 @@ public abstract class RSBackupVaultCmdletBase : RecoveryServicesBackupCmdletBase
         [Parameter(Mandatory = false, HelpMessage = "The Recovery Services Vault.",
             ValueFromPipeline = true)]
         [ValidateNotNullOrEmpty]
-        public ARSVault Vault { get; set; }
+        public string VaultId { get; set; }
+
+        /// <summary>
+        /// Location of the Recovery Services Vault.
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "Location of the Recovery Services Vault.",
+            ValueFromPipeline = true)]
+        [LocationCompleter("Microsoft.RecoveryServices/vaults")]
+        [ValidateNotNullOrEmpty]
+        public string VaultLocation { get; set; }
```

Comment:
Where is this used?

Reply:
This is being used in the Restore cmdlet. You can find this change at https://github.com/Azure/azure-powershell/pull/6327/files#diff-b7f777ff31af1c21fbf6378a2f1e1647